collections/riscv-cpus/index.md

```diff
@@ -0,0 +1,54 @@
+---
+items:
+ - darklife/darkriscv
+ - riscv/riscv-opcodes
+ - freechipsproject/rocket-chip
+ - sifive/freedom
+ - openhwgroup/cv32e40p
+ - lowRISC/ibex
+ - openhwgroup/cva6
+ - csail-csg/riscy-OOO
+ - cornell-brg/lizard
+ - lambdaconcept/minerva
+ - onchipuis/mriscv
+ - SpinalHDL/VexRiscv
+ - roalogic/RV12
+ - syntacore/scr1
+ - SI-RISCV/e200_opensource
+ - lcbcFoo/ReonV
+ - cliffordwolf/picorv32
+ - tomverbeure/mr1
+ - olofk/serv
+ - chipsalliance/Cores-SweRV
+ - chipsalliance/Cores-SweRV-EL2
+ - chipsalliance/Cores-SweRV-EH2
+ - ultraembedded/biriscv
+ - atthecodeface/cdl_hardware
+ - Domipheus/RPU
+ - liangkangnan/tinyriscv 
+ - rsd-devel/rsd
+ - SonalPinto/kronos
+ - stevehoover/warp-v
+ - stnolting/neorv32
+ - rafaelcalcada/steel-core
+ - klessydra/T13x
+ - klessydra/T03x
+ - klessydra/T02x
+ - klessydra/F03x 
+ - stevehoover/RISC-V_MYTH_Workshop
+ - haogwb/starsea_riscv
+ - OSCPU/NutShell
+ - riscv/riscv-qemu
+ - SpinalHDL/VexRiscv
+ - riscv-boom/riscv-boom
+ - riscv/riscv-isa-sim
+ - ultraembedded/riscv
+display_name: Risc-V CPUs
+created_by: carlosdelfino 
+---
+
+The market for processors and microcontrollers is increasingly heated, and with the launch of the ISA (Instruction Set Achitecture) RISC-V, an open specification, it opens up a new opportunity for those who want to act either by researching or collaborating with new processors and microcontrollers.
+
+Those who master the synthesis of Hardware with FPGA, can also propose more concretely new approaches for microcontrollers taking advantage of codes already written for the new architecture.
+
+In this collection I try to present some renowned Colors, and open the opportunity for other colleagues to collaborate with their suggestions.
```

Comment:
Overall this LGTM, but is `Colors` the correct word here? (I'm not super familiar with riscv)